indico/modules/events/persons/templates/management/_person_list_base.html

```diff
@@ -72,6 +72,37 @@
                         </ul>
                     </div>
                 {% endif %}
+                <div class="group">
+                    <a id="no-account-dropdown" href="#" class="i-button arrow highlight disabled js-dropdown" data-toggle="dropdown">
+                        {% trans %}Users with no account{% endtrans %}
+                    </a>
+                    <ul class="dropdown">
+                        <li>
+                            <a href="#"
+                               title="{% trans %}Send an email to all users with no Indico account{% endtrans %}"
+                               data-href="{{ url_for('persons.email_event_persons', event, no_account = True) }}"
+                               data-params-selector="#persons-list [data-no-account] input[name=person_id]:checkbox"
+                               data-title="{% trans %}Send email{% endtrans %}"
+                               data-method="POST"
+                               data-ajax-dialog
+                               data-reload-after>
+                                {% trans %}Email all{% endtrans %}</a>
+                        </li>
+                        <li>
+                            <a id="not-invited-only" href="#"
+                               title="{% trans %}Send an email to all users with no Indico account who have not
+                                     received an invitation email yet{% endtrans %}"
+                               data-href="{{ url_for('persons.email_event_persons', event, no_account = True,
+                                                      not_invited_only = True) }}"
+                               data-params-selector="#persons-list [data-no-account] input[name=person_id]:checkbox"
+                               data-title="{% trans %}Send email{% endtrans %}"
+                               data-method="POST"
+                               data-ajax-dialog
+                               data-reload-after>
+                                {% trans %}Email not invited yet{% endtrans %}</a>
```

Comment:
that text sounds weird. anyone has better ideas on how to label this button? cc @OmeGak  @nop33  @pferreir